Spec/Auth.swift

```diff
@@ -109,20 +109,19 @@ override class var defaultTestSuite : XCTestSuite {
             static let ttl = 1.0
             static let capability = "{\"cansubscribe:*\":[\"subscribe\"]}"
         }
-    override func spec() {
 
-        describe("Basic") {
+        
 
             // RSA1
-            it("should work over HTTPS only") {
+            func test__003__Basic__should_work_over_HTTPS_only() {
```

Comment:
Also notice that I've preserved the indentation level of the original `it` calls. This is to make the diff easy to review. We fix this later in this PR by running a code formatter.